setup.py

```diff
@@ -51,13 +51,13 @@
     "data/CMIP_metrics_results/CMIP5/historical/*.json")
```

Comment:
@gleckler1 @doutriaux1 I've just had a quick look at this and it needs a fairly significant overhaul considering we've now moved to a conda install path.. None of the subdirs make much sense - as in if these are not pulled across in the conda installation there's no need to include them in the setup.py - I'll create a new issue for this